tests/common.py

```diff
@@ -31,6 +31,9 @@
 import opensearch_py_ml as oml
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_PREFIX = "oml_test__"
+INDEX_PREFIX = f"{TEST_PREFIX}_index_"
+MODEL_GROUP_PREFIX = f"{TEST_PREFIX}_model_group_"
```

Comment:
Where are we using this?

Reply:
Actually, I accidentally added this from another ticket. I will remove these.